docs/topics/api/accounts.rst

```diff
@@ -72,7 +72,65 @@ This endpoint is a shortcut to your own account. It returns an :ref:`account obj
 
 .. http:get:: /api/v3/accounts/profile/
 
-    .. _self-account-object:
+
+----------------
+Collections List
```

Comment:
changed to `/api/v3/accounts/account/(int:user_id|string:username)/collections/` in https://github.com/mozilla/addons-server/pull/5358/commits/1aa9eaaf0c0fc9554bdd4f2083e5ef0adae5d3b6